draft-hoffman-dns-over-https-latest.mkd

```diff
@@ -184,26 +184,26 @@ of two different methods:
 GET:
 : Using the on-the-wire representation of a DNS message defined in
 {{RFC1035}} as the query string portion of the URI, encoded with base64url {{RFC4648}},
-using the GET HTTP method. Padding characters for base64url MUST NOT be included. This is the
-preferred approach because it is friendlier to HTTP caches.
+using the GET HTTP method. Padding characters for base64url MUST NOT be included.
+Using GET is the preferred approach because it is friendlier to HTTP caches.
```

Comment:
as mentioned in the other PR, I think we should say what is good about GET and POST rather than expressing a preference.